css/styles.css

```diff
@@ -0,0 +1,72 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  background-color: rgb(244, 244, 244);
+}
+
+.container {
+  width: 100%;
+}
```

Comment:
Is this necessary? I would think it would mean that now your container has no margin on the left or right, which isn't the intent